src/currentprocess/filesource.rs

```diff
@@ -79,71 +79,38 @@ impl StdinSource for super::TestProcess {
     }
 }
 
-// -------------- stdout -------------------------------
-
-pub trait Isatty {
-    fn isatty(&self) -> bool;
-}
-
-/// Stand-in for std::io::StdoutLock
+/// Allows you to write to a terminal or other output stream.
 pub trait WriterLock: Write {}
 
-/// Stand-in for std::io::Stdout
-pub trait Writer: Write + Isatty + Send {
+/// Holds a terminal or other output stream.
+pub trait Writer: Send + Sync {
+    /// Locks the terminal so that you can write to it.
     fn lock(&self) -> Box<dyn WriterLock + '_>;
 }
 
-/// Stand-in for std::io::stdout
+// -------------- stdout -------------------------------
```

Comment:
Why the change from a docstring to a mere comment?

Reply:
Because the original comment said it is `Stand-in` instead of `Stand-out`. I guess it is a typo.

Reply:
Reverted! Thanks!